src/selectize.js

```diff
@@ -231,7 +231,8 @@ $.extend(Selectize.prototype, {
 					return false;
 				}
 				// blur on click outside
-				if (!self.$control.has(e.target).length && e.target !== self.$control[0]) {
```

Comment:
$control will never have e.target here, as e.target is contained within $dropdown.  This appears to be a logic error.  When clicking inside of the dropdown we do not want to blur, so $dropdown.has(e.target) is the correct logic here.